docs/src/components/Combination.js

```diff
@@ -15,6 +15,7 @@ type Props = {
   showHeading?: boolean,
   showValues?: boolean,
   stacked?: boolean,
+  useLabelForValues?: boolean,
```

Comment:
@ayeshakmaz instead of making this a boolean prop, can we pass in a `labelPrefix`? Using a boolean results in only being able to use 1 `Combination` with `useLabelForValues` on a page. If we pass something like `radiobutton-size-example` to `labelPrefix`, we can have multiple `Combination` elements with `useLabelForValues` on a page.

Reply:
ahhhh yes, great call - I was also thinking of using the ID, but this will give us the most flexibility, I'll make that update!